torch/nn/modules/module.py

```diff
@@ -1265,6 +1265,8 @@ def train(self: T, mode: bool = True) -> T:
         mode, if they are affected, e.g. :class:`Dropout`, :class:`BatchNorm`,
         etc.
 
+        Additionally, consider using :class:`~no_grad` when evaluating a model.
```

Comment:
I am not sure the link here will work properly as `no_grad` is not in the `torch.nn` namespace.

Reply:
Do you have any ideas about what it should be?

Reply:
By building locally, doing ```:class:`torch.no_grad` ``` does generate the right link.